Dockerfile.openhabian-aarch64-64bit

```diff
@@ -1,7 +1,7 @@
 # Note RPi3 base (first RPi to have aarch64 available) WITH 64bit OS
 #FROM balenalib/raspberrypi3-debian-node:latest-buster
 #FROM balenalib/generic-aarch64-debian-node
-FROM balenalib/raspberrypi4-64-debian-node-build
+FROM balenalib/raspberrypi4-64-debian-node:latest-buster-build
```

Comment:
Never know why you change between different images for the CI. Seems unrelated to the topic of the PR...
But still ok for me.

Reply:
Yes this is unrelated Sorry I should have separated that. This was meant to just be a test because one CI test in this PR continously failed because of that, then again now that it works I'll put it in.